src/meetingreadinesschecker/DefaultMeetingReadinessChecker.ts

```diff
@@ -59,15 +61,17 @@ export default class DefaultMeetingReadinessChecker implements MeetingReadinessC
   }
 
   async checkAudioOutput(
-    audioOutputDeviceInfo: MediaDeviceInfo,
+    audioOutputDeviceInfo: MediaDeviceInfo | string,
```

Comment:
Again.

Reply:
ChooseAudioOutput API actually only accepts string and not Device. We do not use it here but using string would be consistent